app/core/config.py

```diff
@@ -17,8 +17,8 @@ class Settings(BaseSettings):
 
     # CORS_ORIGINS is a JSON-formatted list of origins
     CORS_ORIGINS: List[AnyHttpUrl] = [
-        "http://localhost",
-        "https://localhost",
+        "http://localhost:3000",
+        "https://localhost:3000",
```

Comment:
@arthur-ruxton this is the default configuration for the API. I just needed to re-add port 3000 so that cross-origin requests work locally.